src/frontend/src/store/index.ts

```diff
@@ -411,6 +423,20 @@ export default new Vuex.Store({
         )
     },
 
+    ACTION_START_HISTORY_INTERVAL({ commit, getters, dispatch }) {
+      const interval = window.setInterval(() => {
+        if (getters.umnetId.length > 0) { 
+          dispatch("ACTION_FETCH_TRANSACTION_HISTORY");
+        }
+      }, 10000);
+
+      commit('MUTATION_SET_HISTORY_INTERVAL', interval)
+    },
+
+    ACTION_CLEAR_HISTORY_INTERVAL({ commit }) {
```

Comment:
I think you can just call the mutation wherever you need it, instead of calling an action that calls a mutation